apps/next-evans/components/pages/home/FeaturedBooksBlock.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react';
+import cx from 'classnames';
+import { Swipeable } from 'react-swipeable';
+import { t } from '@friends-library/locale';
+import type { DocumentWithMeta } from '@/lib/types';
+import FeaturedBook from './FeaturedBook';
+import Heading from '@/components/core/Heading';
+import WoodgrainSVG from '@/public/images/woodgrain.svg';
+
+interface Props {
+  books: DocumentWithMeta[];
+}
+
+const FeaturedBooksBlock: React.FC<Props> = ({ books }) => {
+  const [index, setIndex] = useState<number>(0);
+  const [controlled, setControlled] = useState<boolean>(false);
+
+  useEffect(() => {
+    if (controlled) return;
+    const timeout = setTimeout(() => {
+      setIndex(index === books.length - 1 ? 0 : index + 1);
+    }, 6500);
+    return () => clearTimeout(timeout);
+  });
+
+  return (
+    <Swipeable
+      nodeName="section"
+      className="py-10 sm:py-12 md:py-20 bg-[#f9f9f9] [background-sie:cover] md:[background-size:200%] lg:[background-size:150%] bg-center"
+      style={{ backgroundImage: `url(${WoodgrainSVG.src})` }}
+      onSwipedRight={() => {
+        setControlled(true);
+        setIndex(index === 0 ? books.length - 1 : index - 1);
+      }}
+      onSwipedLeft={() => {
+        setControlled(true);
+        setIndex(index === books.length - 1 ? 0 : index + 1);
+      }}
+    >
+      <Heading className="text-gray-800">{t`Featured Books`}</Heading>
+      <div className="BooksViewer w-[calc(100vw-(100vw-100%))] overflow-hidden">
```

Comment:
can you do a slow once-over of the changes in this PR looking for classes like `BooksViewer` that aren't tailwind utilities and checking if we can delete them? i'd like to avoid transferring classes that aren't doing anything to the new site if possible.